lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -741,6 +807,19 @@ class HybridTextEditing {
       ..transform = transformCss;
   }
 
+  // TODO(flutter_web): When the browser closes and re-opens the virtual
+  // shifts the page in IOS. Call this method from visibility change listener
```

Comment:
nit: it's spelled iOS (applies here and elsewhere in this PR)

Reply:
Done.